### DISCUSSION TITLE
Two transitions next to each other

### DISCUSSION THREAD

**Discussion by stephanebertrand** [AcademySoftwareFoundation/OpenTimelineIO#1557] [Q&A] [answered]
Hi,

i read in documentation Timeline Structure / Transition
_"Also, you cannot place two transitions next to each other in a track; there must be a composable item between them."_
Indeed, an exception is raised by _expand_transition in track_algo.py

But i don't understand "**why**".
You can have a clip that fades to black. And just after, without a gap, a clip that opens from black?

In other words, I can't find a configuration where this is a problem. But I suspect that this test is not done for nothing.

Thank you for your clarification

**Comment by jminor** [comment, marked as answer]
In the case you described you should add a 0-duration black clip (with a generator reference indicating the color) between the two transitions. That way you are able to define the color to fade to/from in that clip. Without that composable object, the transition wouldn't know what media to fade to/from.

Alternately you could use a single transition between your two clips, and specify that the transition is a "dip to black" effect instead of a cross-dissolve. OTIO doesn't yet have a formal way to specify the type of effect used in the transition though, so depending on your workflow that might not be an option.
